priorityq.py

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env python
+# Priority Queue Python file for Efrain, Mark and Henry
+import queue as q
+
+
+class Pq(object):
+    """Initiate priority queue"""
+
+    def __init__(self, num=3):
+        """Initiate pq with optional priority range, default is 3."""
+        for x in range(1, num+1):
+            setattr(self, 'que{}'.format(x), q.Queue())
+        self.num_queues = num
```

Comment:
Why not simply set something like this:

```python
self.queues = [q.Queue() for i in range(num)]
```

Then you could simply treat the incoming priority as an index into the list of queues.